source/NVDAObjects/window/winword.py

```diff
@@ -1430,10 +1466,520 @@ def _get_WinwordWindowObject(self):
+		self.keyList.append('otherElements')
+		self.elementsCount=len(self.keyList)
+		super(WordChart,self).__init__(windowHandle=windowHandle)
+
+	def focusSeriesTrendline(self, seriesIndex):
+		obj=WordChartSeriesTrendline(windowHandle=self.windowHandle, wordApplicationObject=self.wordApplicationObject, wordChartObject=self.wordChartObject, keyIndex=self.keyIndex, seriesIndex=seriesIndex)
+		eventHandler.queueEvent('gainFocus',obj)
+
+	def focusChartSeries(self, seriesIndex):
+		self.currentSeriesIndex=seriesIndex
+		obj=WordChartSeries(windowHandle=self.windowHandle, wordApplicationObject=self.wordApplicationObject, wordChartObject=self.wordChartObject, keyIndex=self.keyIndex, seriesIndex=self.currentSeriesIndex)
+		self.wordChartObject.SeriesCollection(self.currentSeriesIndex).Select()
+		eventHandler.queueEvent('gainFocus',obj)
+
+	def focusChartElements(self):
+		obj=WordChartElement(windowHandle=self.windowHandle, wordApplicationObject=self.wordApplicationObject, wordChartObject=self.wordChartObject, keyIndex=self.keyIndex)
+		eventHandler.queueEvent('gainFocus',obj)
+
+	def _get_name(self):
+		if self.wordChartObject.HasTitle:
+			name=self.wordChartObject.ChartTitle.Text
+		else:
+			name=self.wordChartObject.Name
+		#find the type of the chart
+		chartType = self.wordChartObject.ChartType
+		chartTypeText = _chartConstants.chartTypeDict.get(chartType,
+                # Translators: Reported when the type of a chart is not known.
+                                _("unknown"))
+		# Translators: Message reporting the title and type of a chart.
+		text=_("Chart title: {chartTitle}, type: {chartType}").format(chartTitle=name, chartType=chartTypeText)
+		count = self.wordChartObject.SeriesCollection().count
+		if count>0:
+                        if count == 1:
+                                # Translators: Indicates that there is 1 series in a chart.
+                                seriesValueString = _( "There is 1 series in this chart" )
+                        else:
+                                # Translators: Indicates the number of series in a chart where there are multiple series.
+                                seriesValueString = _( "There are total %d series in this chart" ) %(count)
+                        for i in xrange(1, count+1):
+                                # Translators: Specifies the number and name of a series when listing series in a chart.
+                                seriesValueString += ", " + _("series {number} {name}").format(number=i, name=self.wordChartObject.SeriesCollection(i).Name)
+                        text += seriesValueString
+                else:
+                        # Translators: Indicates that there are no series in a chart.
+                        text +=_("No Series defined.")
+                return text
+
+	def _get_title(self):
+		try:
+			title=self.wordChartObject.ChartTitle.Text
+		except COMError:
+			title=None
+		return title
+
+	def _get_role(self):

```

Comment:
Sorry! I just didn't think about that while coding.
Changed code is in branch in_t1541_Fin
Thanks a lot for the code review.